File: AAC-Practice/app/src/main/java/com/bowoon/android/aac_practice/Memo.java
package com.bowoon.android.aac_practice;

import org.joda.time.DateTime;

import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "memo")
public class Memo {
    @PrimaryKey(autoGenerate = true)
    private int id;
    @ColumnInfo(name = "content")
    private String content;
    @ColumnInfo(name = "time")
    private DateTime time;

    public Memo(String content) {
        this.content = content;
        this.time = new DateTime(System.currentTimeMillis());
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getContent() {
        return content;
    }

    public void setContent(String content) {
        this.content = content;
    }

    public DateTime getTime() {
        return time;
    }

    public void setTime(DateTime time) {
        this.time = time;
    }
}

File: AACPracticeKotlin/app/src/main/java/com/bowoon/android/aac_practice_kotlin/Memo.kt
package com.bowoon.android.aac_practice_kotlin

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey
import org.joda.time.DateTime

@Entity(tableName = "memo")
data class Memo(
    @ColumnInfo(name = "content")
    var content: String?,
    @PrimaryKey(autoGenerate = true)
    var id: Int = 0,
    @ColumnInfo(name = "time")
    var time: DateTime = DateTime(System.currentTimeMillis())
)